src/unistd.rs

```diff
@@ -2240,29 +2247,63 @@ impl Group {
 }
 
 #[derive(Debug)]
-/// Query a group. The buffer size variants should not be needed 99% of the time; the default
-/// buffer size of 1024 should be more than adequate. Only use the buffer size variants if you
-/// receive an ERANGE error without them, or you know that you will be likely to.
+/// Query a user.
+///
+/// The buffer size variants should not be needed 99% of the time; the default buffer size of 1024
+/// should be more than adequate. Only use the buffer size variants if you receive an ERANGE error
+/// without them, or you know that you will be likely to.
+///
+/// # Example
```

Comment:
s/Example/Examples for consistency with the rest of the project

Reply:
@asomers What consistency? `mkfifo`, `getcwd` etc use "Example"...this makes sense because it's a lone example, in their case as well as mine.

Reply:
I'm not saying we're perfect, but currently Nix has 30 "Examples" compared to 9 "Example".  Best stick to the predominant style.

Reply:
@asomers OK